test/utils.el

```diff
@@ -0,0 +1,23 @@
+(require 'dash)
+
+(defmacro codemetrics-test (test-name file major-mode expected-result)
+  "Create a codemetrics test named TEST-NAME using FILE in MAJOR-MODE.
+Expecting result EXPECTED-RESULT."
+  `(ert-deftest ,test-name ()
+     (with-current-buffer (find-file-noselect ,file)
+       (,major-mode)
+       (tree-sitter-mode)
+       (let* ((codemetrics-results (codemetrics-buffer))
+              (total-score (car codemetrics-results))
+              (expressions-and-scores (codemetrics-test-utils-expression-scores codemetrics-results)))
+         (should (equal ,expected-result
+                        (cons total-score
+                              expressions-and-scores)))))))
+
+(defun codemetrics-test-utils-expression-scores (analyze-result)
+  "Helper method to get a list of simple expression + score pairs to work with."
+  (--map (cons (tsc-node-type (car it))
+               (nth 2 it))
+         (cdr analyze-result)))
+
+(provide 'utils)
```

Comment:
We can rename this file to `test-helper.el`, then get rid of this line and all `(require 'util)` (in other test files). :)

Reference: https://github.com/rejeep/ert-runner.el/blob/master/ert-runner.el#L205

Reply:
Wow! Did not know it worked like that. Learned something new now, and the tests are way more clear and to-the-point now 😄 Thanks for the tip! Done in latest commits.